node/pkg/watchers/evm/connectors/celo.go

```diff
@@ -181,6 +182,19 @@ func (c *CeloConnector) RawCallContext(ctx context.Context, result interface{},
 	return c.rawClient.CallContext(ctx, result, method, args...)
 }
 
+func (c *CeloConnector) RawBatchCallContext(ctx context.Context, b []ethRpc.BatchElem) error {
+	celoB := make([]celoRpc.BatchElem, len(b))
+	for i, v := range b {
```

Comment:
Why doesn't this look like the ethereum.go version?

Reply:
Because it has to convert an ethereum struct to a celo struct.